src/instruments/view/instrumentlistmodel.cpp

```diff
@@ -38,14 +38,16 @@ static const QString GROUP_ID("groupId");
 static const QString CONFIG_KEY("config");
 static const QString SOLOIST_KEY("isSoloist");
 static const QString PART_KEY("isExistingPart");
+static const QString SOLOISTS_SCOREORDERGROUP("<soloists>");
+static const QString UNSORTED_SCOREORDERGROUP("<unsorted>");
```

Comment:
Probably those strings are unused

Reply:
Indeed a relic of previous version. Removed.